mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -55,6 +62,40 @@ class DistanceSensorItem {
 
 	void range_cb(const sensor_msgs::Range::ConstPtr &msg);
 	static Ptr create_item(DistanceSensorPlugin *owner, std::string topic_name);
+
+private:
+	std::vector<float> data;	//!< array allocation for measurements
+	size_t data_index;			//!< array index
+
+	/**
+	 * Calculate measurements variance to send to the FCU.
+	 */
+	float calculate_variance(float range) {
+		if (data.size() < 50)	// limits the size of the array to 50 elements
+			data.push_back(range);
+		else {
+			data[data_index] = range;	// it starts rewriting the values from 1st element
+			if (++data_index > 49) data_index = 0;	// restarts the index when achieves the last element
+		}
+
+		float average, variance, sum = 0, sum_ = 0;
+
+		/*  Compute the sum of all elements */
+		for (auto i : data)
```

Comment:
Second style: remove unneeded `{}` braces, or at least remove new line after for.